### PR TITLE
Simplify ParquetPageSource

### PR DESCRIPTION
**Pull request by linxingyuan1102**
Please see the commit message for explanations.

This is preparatory work for decoupling `ParquetPageSource` from `HiveColumnHandle` and removing `HiveColumnHandle` in Iceberg Connector (#1324 )

### CODE REVIEW

File: presto-hive/src/main/java/io/prestosql/plugin/hive/parquet/ParquetPageSource.java [hunk 1]
```diff
@@ -146,28 +130,12 @@ public Page getNextPage()
...
             for (int fieldId = 0; fieldId < blocks.length; fieldId++) {
-                if (constantBlocks[fieldId] != null) {
-                    blocks[fieldId] = constantBlocks[fieldId].getRegion(0, batchSize);
-                }
-                else {
-                    Type type = types.get(fieldId);
-                    Optional<Field> field = fields.get(fieldId);
-                    int fieldIndex;
-                    if (useParquetColumnNames) {
-                        fieldIndex = getFieldIndex(fileSchema, columnNames.get(fieldId));
```
Comment (electrum) [inline presto-hive/src/main/java/io/prestosql/plugin/hive/parquet/ParquetPageSource.java]: This method is now unused and can be removed.

Comment (linxingyuan1102) [reply presto-hive/src/main/java/io/prestosql/plugin/hive/parquet/ParquetPageSource.java]: Do you mean `getFieldIndex`? I've removed it in the commit.

File: presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergPageSourceProvider.java [hunk 2]
```diff
@@ -65,7 +65,6 @@
 import java.util.Objects;
 import java.util.Optional;
 import java.util.OptionalInt;
-import java.util.Properties;
```
Comment (electrum) [inline presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergPageSourceProvider.java]: This change should be in the previous commit

File: presto-hive/src/main/java/io/prestosql/plugin/hive/parquet/ParquetPageSource.java (hunk 1 above)
Comment (electrum) [reply presto-hive/src/main/java/io/prestosql/plugin/hive/parquet/ParquetPageSource.java]: I mean the method can be deleted from `ParquetTypeUtils`. Perhaps you already removed it locally but didn't push to GitHub?

Comment (linxingyuan1102) [reply presto-hive/src/main/java/io/prestosql/plugin/hive/parquet/ParquetPageSource.java]: Oh I see, I didn't remove it from `ParquetTypeUtils`. I thought you meant the call of this method. Will do.

**Comment by linxingyuan1102** [comment]
@electrum Thanks for the review. Comments addressed

**Comment by electrum** [comment]
Looks good. Will merge after Travis passes.

**Comment by electrum** [comment]
There's a checkstyle error:
```java
[INFO] --- maven-checkstyle-plugin:3.0.0:check (checkstyle) @ presto-parquet ---
[INFO] There are 2 errors reported by Checkstyle 8.16 with checkstyle/airbase-checks.xml ruleset.
[ERROR] src/main/java/io/prestosql/parquet/ParquetTypeUtils.java:[22,8] (imports) UnusedImports: Unused import - org.apache.parquet.io.InvalidRecordException.
[ERROR] src/main/java/io/prestosql/parquet/ParquetTypeUtils.java:[32,8] (imports) UnusedImports: Unused import - java.util.Locale.
```

**Comment by linxingyuan1102** [comment]
> There's a checkstyle error

Oh I forgot to check this. Just updated it. Now `mvn clean install -DskipTests` builds successfully in `presto-parquet`.

**Comment by electrum** [comment]
Sometimes IntelliJ will forget to update the imports, even when *Optimize imports on the fly* is enabled. It's very annoying and a big time waster.

**Pull request merged**